wagtail_footnotes/blocks.py

```diff
@@ -25,6 +27,15 @@ def __init__(self, **kwargs):
         if "footnotes" not in self.features:
             self.features.append("footnotes")
 
+    def render_footnote_tag(self, index):
+        template_name = getattr(
+            settings,
+            "WAGTAIL_FOOTNOTES_REFERENCE_TEMPLATE",
+            "wagtail_footnotes/includes/footnote_reference.html"
+        )
+        template = get_template(template_name)
+        return template.render({"index": index})
```

Comment:
I like the template approach much better than f-strings.

At first I was in two minds about adding a setting over than just the usual pattern of overriding templates from other apps by adding the correct path in your project's templates. 
In this case, adding `wagtail_footnotes/includes/footnote_reference.html` to one's app's templates folder, or a top-level templates folder should override it.

On the other hand, Django, Wagtail and some packages do provide a setting for specific templates too (like the [password required template](https://docs.wagtail.org/en/stable/advanced_topics/privacy.html#setting-up-a-global-password-required-page) in Wagtail), so this makes it more expressive

Reply:
Yeah. I thought about that too. There's nothing with this approach that *prevents* just overriding `wagtail_footnotes/includes/footnote_reference.html`. Do you think there be a note about that option in the README?

Reply:
No, I think we should guide people towards the setting as the official way of doing it